src/_findErroneousUnionNode.ts

```diff
@@ -10,16 +10,35 @@ import { _doesCrossjoinRepresentAnExpandedMember } from "./_doesCrossjoinReprese
 import { _doesCrossjoinYieldAllCombinationsOfMembers } from "./_doesCrossjoinYieldAllCombinationsOfMembers";
 
 /**
- * The shape of an erroneous union node.
- * Erroneous union nodes can lead to errors and must be cleaned up.
+ * An Mdx union function whose arguments are all crossjoins.
  */
-export interface ErroneousUnionNode extends MdxFunction {
+export interface UnionNodeWithCrossjoinsAsArguments extends MdxFunction {
   name: "Union";
   arguments: (MdxFunction & {
     name: "Crossjoin";
   })[];
 }
 
+/**
+ * A node matched during an Mdx traversal process, and the path leading to it within the traversed mdx.
+ */
+export type MatchAndPath<T extends Mdx> = {
```

Comment:
This type would be useful in ActiveUI as well: for all places using [findDescendant](https://github.com/activeviam/activeui/blob/master/packages/mdx/src/findDescendant.ts).